packages/web/src/services/audio-player/AudioPlayer.ts

```diff
@@ -370,6 +374,9 @@ export class AudioPlayer {
   }
 
   pause = () => {
+    if (this.gainNode) {
+      this.gainNode.gain.setValueAtTime(1, 0)
+    }
```

Comment:
Disclaimer: I still don't fully understand the purpose of this call, so feel free to ignore this suggestion...

All the [examples](https://developer.mozilla.org/en-US/docs/Web/API/BaseAudioContext/createGain#examples) I've seen are using currentTime to set the gain.

To me this makes more sense, since in the other place we call `setValueAtTime`, we're inside a condition that guarantees currentTime is 0:
`this.gainNode.gain.setValueAtTime(1, this.audioCtx?.currentTime ?? 0)`

I tried both, and it was hard to tell the difference, but I think the fade was less severe on the currentTime version. I'm still also confused about the currentTime not moving when we skip issue. Is that still happening?

Reply:
I think your change is correct

Reply:
I'm still confused about it too. It does still happen, but I don't think it's actively breaking anything. Worth a follow up investigation at some point, probably, but I think ok to leave for now